scripts/deploy_local.sh

```diff
@@ -485,10 +485,11 @@ install_dependencies() {
       packages_to_install=("${packages_to_install[@]/folly}")
     fi
 
-    if [[ "${packages_to_install[@]}" =~ "zookeeper" ]]; then
-      install_zookeeper=true  # set zookeeper install flag
-      # remove zookeeper from packages_to_install
-      packages_to_install=("${packages_to_install[@]/zookeeper}")
+    if [[ "${packages_to_install[@]}" =~ "zetcd" ]]; then
+      log "Installing zetcd."
+      go get github.com/etcd-io/zetcd/cmd/zetcd
```

Comment:
Have we guaranteed that `go` has been installed?

Reply:
the go is install in previous step